src/index.js

```diff
@@ -1,123 +1,111 @@
 const TESTING = global.COMB !== undefined;
-
-if (!TESTING)
+if (!TESTING) {
   COMB = require('@holo-host/comb').COMB;
+}
 
 const { EventEmitter } = require('events');
 
-function makeUrlAbsolute (url) {
+function makeUrlAbsolute(url) {
   return new URL(url, window.location).href
 }
 
-class Connection extends EventEmitter {
-
-  constructor(url, signalCb, opts) {
+  // Child Msg Bus Alert Types:
+  // - `sign-in` - emitted when the user completes a successful sign-in
+  // - `sign-up` - emitted when the user completes a successful sign-up
+  // - `sign-out` - emitted when the user competes a successful sign-out
+  // - `canceled` - emitted when the user purposefully exits sign-in/up
+  // - `signal` - emitted when a signal is passed from chaperone
+  // - `available` - emitted when the WebSDK to chaperone and envoy are opened and hosted app is ready for zome calls
+  // - `unavailable` - emitted when the ws WebSDK to chaperone and/or envoy is closed
+  // - `unrecoverable-agent-state` - emitted when an unrecoverable error event is passed from chapeone
+
+class WebSdkApi extends EventEmitter {
+  constructor(child) {
     super();
+    this.available = null
+    this.child = child;
+    child.msg_bus.on("alert", (event, ...args) => this.emit(event));
```

Comment:
I think it also needs to forward the specific events
```suggestion
    child.msg_bus.on("alert", (event, ...args) => this.emit(event))
    child.msg_bus.on("signal", signal => this.emit('signal', signal))
    child.msg_bus.on("available", () => this.emit('available'))
    child.msg_bus.on("unavailable", () => this.emit('unavailable'))
    child.msg_bus.on("unrecoverable-agent-state", () => this.emit('unrecoverable-agent-state'))
```